### PR TITLE
fix and update tests on get-n-parents after logic update

### PR DESCRIPTION
**Pull request by ewong26**
PR to update tests after #78 so they don't fail on deploy.

How to test:
* In terminal run: ` jest test/unit/modules/helpers/get-n-parents-spec.js`

### CODE REVIEW

File: test/unit/modules/helpers/get-n-parents-spec.js [hunk 1]
```diff
@@ -28,7 +27,113 @@ describe('RecordingLibrary.helpers.getNParents', () => {
...
+    p4WithAnchorTag.setAttribute('href', '#foo');
+    p4WithAnchorTag.setAttribute('type', 'submit');
+
+    const pSection = document.createElement('section');
+    pSection.setAttribute('class', 'parent-section');
+
+
+    p5.removeChild(p4);
+    p5.appendChild(pSection);
+    pSection.appendChild(p4WithAnchorTag);
```
Comment (ewong26) [inline test/unit/modules/helpers/get-n-parents-spec.js:134]: I swapped the label tag here with an anchor tag so we can test the new logic + added in an extra parent so we simulate the full recursive effects when there is a tagName or role attribute that requires us to collect additional parents.

Comment (ewong26) [reply test/unit/modules/helpers/get-n-parents-spec.js:134]: if we wanted to, we could add another test that uses a role attribute that triggers the new logic but that seems a bit redundant to me.

**Review by pevner-p2 — APPROVED** [review]

**Review by jongsch — APPROVED** [review]

File: test/unit/modules/helpers/get-n-parents-spec.js [hunk 2]
```diff
@@ -28,7 +27,113 @@ describe('RecordingLibrary.helpers.getNParents', () => {
...
+        text: null,
+        title: 'h1title',
+        type: null,
+        href: null,
+        nth_parent: 5,
+      },
+    ]);
+  });
+
+  test(`should return element attributes up 9 parents away instead of 5 parents away because:
```
Comment (dabrady) [inline test/unit/modules/helpers/get-n-parents-spec.js:119]: [not blocking] Huh, I thought we were only intending to grab a total of 7 🤔 But looking at the [code](https://github.com/ProdPerfect/prodperfect-keen-tracking.js/pull/78/files), we actually don't put an upper bound on it.

**Review by dabrady — APPROVED** [review]
Makes sense to me man, thanks for this!

**Comment by dabrady** [comment]
By the way none of the CI checks on this PR will pass: the Semaphore linkage is borked on this repo for some reason, and we stopped paying for Hound. 😆 🔥 

Your build is here: https://semaphoreci.com/prodperfect/prodperfect-keen-tracking-js/branches/update-get-n-parents-test/builds/1

File: test/unit/modules/helpers/get-n-parents-spec.js (hunk 2 above)
Comment (ewong26) [reply test/unit/modules/helpers/get-n-parents-spec.js:119]: yea, I think we just keep looping and looping and looping so if there are two buttons or a button and an anchor tag... lolz

**Pull request merged**